libbeat/autodiscover/providers/kubernetes/config.go

```diff
@@ -30,6 +30,9 @@ import (
 	"github.com/elastic/elastic-agent-libs/logp"
 )
 
+// AllSupportedHints includes the set of all supported hints for both logs and metrics autodiscovery
+var AllSupportedHints = []string{"enabled", "module", "metricsets", "host", "period", "timeout", "metrics_path", "username", "password", "stream", "processors", "multiline", "json", "disable", "ssl", "metrics_filters", "raw", "include_lines", "exclude_lines", "fileset", "pipeline", "raw"}
```

Comment:
should it be `host` or `hosts`? https://www.elastic.co/guide/en/beats/metricbeat/current/configuration-autodiscover-hints.html#_co_elastic_metricshosts

Reply:
Nice catch @tetianakravchenko . In beats it is "hosts"

Although in elastic-agent should be host 
(https://github.com/elastic/elastic-agent/blob/main/internal/pkg/composable/providers/kubernetes/hints.go#L49)

Reply:
I updated in [55451c6](https://github.com/elastic/beats/pull/38213/commits/55451c69efd4e9a1c7dc7cb7d922a6ec9bfe8c21) and also added some test